docker/Dockerfile.py

```diff
@@ -34,6 +34,12 @@
 
     RUN ln -s /home/s3am/bin/s3am /usr/local/bin/
 
+    RUN sudo apt-get -y remove docker-engine
+
+    RUN wget -O /usr/bin/docker https://get.docker.com/builds/Linux/x86_64/docker-1.10.3
```

Comment:
This is a big change. This should have been a separate commit or the commit message should be adjusted.